src/bioregistry/resolve_identifier.py

```diff
@@ -468,6 +468,12 @@ def get_iri(
     A custom provider is given, which makes the Bioregistry very extensible
     >>> get_iri("chebi:24867", provider="chebi-img")
     'https://www.ebi.ac.uk/chebi/displayImage.do?defaultImage=true&imageIndex=0&chebiId=24867'
+
+    An override for a metaregistry is built-in to the record
+    >>> get_iri("kisao:0000057", provider="ols")
+    'https://www.ebi.ac.uk/ols/ontologies/kisao/terms?iri=http://www.biomodels.net/kisao/KISAO#KISAO_0000057'
```

Comment:
should be able to get base IRI from ols, however, bioportal is a completely different situation